### DISCUSSION TITLE
Launching a MP modal without the initialisation of the button.

### DISCUSSION THREAD

**Discussion by VladLegkowski** [mercadopago/sdk-js#191] [General]
Hi, is there a way to launch a modal without clicking on **_Pay with Mercado Pago_** button?

```
await bricksBuilder.create(
            'wallet',
            this.mercadoPagoContainerId,
            {
                initialization: {
                    preferenceId: this.preferenceId,
                    redirectMode: 'modal'
                }
            }
        )
```

Basically `.createAndLaunch` of any sort?

As far as I see based on the documentation, the standard way to work with Mercado Pago involves using a button to initiate a transaction.

**Comment by francineemilia** [comment]
Hi @VladLegkowski,

Unfortunately, there is currently no way to open the modal without clicking.

I remain available if you have any questions,

**Reply by VladLegkowski** [reply]
Thanks @francineemilia, thank you for the quick response!
